gpu-operator/getting-started.rst

```diff
@@ -406,6 +406,21 @@ options are used with the container-toolkit deployed with GPU Operator:
       - name: CONTAINERD_SET_AS_DEFAULT
         value: true
 
+These options can be passed to GPU Operator during install time as below.
```

Comment:
I agree that the syntax for the options can be tricky.  Do you mind revising the intro statement so the purpose is to clarify the syntax?  Something like the foll?

If you need to specify custom values, refer to the following sample command for the syntax: